<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_songPath"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lb_songPath">
          <property name="text">
           <string>Song Path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lb_lyricPath">
          <property name="text">
           <string>Lyric Path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pb_lyricPath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_lyricPath"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pb_songPath">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pb_sure">
          <property name="text">
           <string>Sure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>About author</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>界面美化：张佳琦</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>版权所有，盗版必究</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>代码实现：张帅</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>版本号：1.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>groupBox_2</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Startup item</string>
     </property>
     <widget class="QCheckBox" name="cb_close">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimize to pallets when closed</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_welcome">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start greeting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_set">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>set</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
